client/components/checkbox/index.jsx

```diff
@@ -0,0 +1,74 @@
+/**
+* External dependencies
+*/
+import React, { PropTypes } from 'react';
+import classNames from 'classnames';
+
+/**
+* Interal dependencies
+*/
+import Gridicon from 'components/gridicon';
+
+export default React.createClass( {
+	displayName: 'Checkbox',
+
+	propTypes: {
+		checked: PropTypes.bool.isRequired,
+		disabled: PropTypes.bool,
+		onChange: PropTypes.func
+	},
+
+	getInitialState() {
+		return {
+			focused: false
+		}
+	},
+
+	getDefaultProps() {
+		return {
+			disabled: false
+		};
+	},
+
+	handleChange( e ) {
+		const { onChange } = this.props;
+
+		onChange && onChange( e, this.state.checked );
```

Comment:
`this.state.checked` is passed as a parameter here but doesn't exist. did you mean to send `this.props.checked`

actually, I think we could simplify this API to send only a single boolean value - the new checked status. that is, if we have a checkbox that is already checked, we should send out `false` when clicked to indicate the new status.

```js
toggleCheckmark() {
	const { checked, onChange } = this.props;

	onChange( ! checked );
}
```